src/ogs.styl

```diff
@@ -139,6 +139,11 @@ light.weak-loss                         = lighten(light.loss, 25%)
 light.strong-win                        = darken(light.win, 25%)
 light.strong-loss                       = darken(light.loss, 25%)
 
+light.strong-w-text                     = #E4DCDC
+light.strong-l-text                     = #E4DCDC
+light.weak-w-text                       = #3C3939
+light.weak-l-text                       = #3C3939
```

Comment:
The text color on the page is mostly #202020

Reply:
Color updated to #202020